### PR TITLE
Android changes

### PR DESCRIPTION
**Pull request by Kanchan-Salotra-brightcove**
Geo restriction changes applied .
image

### CODE REVIEW

**Review by backer — COMMENTED** [review]
this looks good to me but waiting on our test account to be updated with playback rights so that we can verify

File: example/android/app/src/main/java/com/example/MainActivity.java [hunk 1]
```diff
@@ -12,4 +14,9 @@ public class MainActivity extends ReactActivity {
   protected String getMainComponentName() {
     return "example";
   }
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
```
Comment (backer) [inline example/android/app/src/main/java/com/example/MainActivity.java]: for PiP, let's create a `BrightcoveReactActivity` that extends `ReactActivity` and includes the necessary callbacks to support PiP. Then the `example/MainActivity` can extend that in order to support it in the react native application

**Comment by backer** [comment]
@Kanchan-Salotra-brightcove when using png's for image resources, we need to include versions in each image set (mdpi through xxhdpi). Also, I see that you have been force-pushing to this branch, which erases the commit history for it each time. Please push normally moving forward, thanks!

**Comment by Kanchan-Salotra-brightcove** [comment]
@backer ,
all the png icons are added ,Please check.

**Comment by backer** [comment]
@Kanchan-Salotra-brightcove I tried pulling and running to check the PiP behavior but I'm running into build errors. Have you dev tested to confirm that everything is working properly?

**Comment by Kanchan-Salotra-brightcove** [comment]
hi @backer , 

Yes i have rechecked all is working fine. as per the expectation.

**Review by backer — CHANGES_REQUESTED** [review]
looking over the image assets, we shouldn't be using a chevron for the PiP button. I recommend using the ootb player button ui even if we use custom click/visibility logic. Alternatively, we could use an image more representative of PiP, like the attached SVG.


Group 3366

**Review by backer — APPROVED** [review]
👍

**Pull request merged**